src/containers/search/SearchResults.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import getSearchData from "../../actions/getSearchDataAction";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+export const onClickHandler = (id, articleAuthor) => event => {
+  localStorage.setItem("articleId", id);
+  localStorage.setItem("articleAuthor", articleAuthor);
+};
+
+export class SearchResults extends Component {
+  componentDidMount() {
+    this.props.getSearchData(
+      this.props.match.params.type,
+      this.props.match.params.searchData
+    );
+  }
+  articleDiv = (id,name,title) => {
```

Comment:
Could you make this function more descriptive

Reply:
@shakirandagire I have worked on this.